repos/fdbt-site/src/components/OperatorGroupCard.tsx

```diff
@@ -24,6 +24,7 @@ const OperatorGroupCard = ({
                         <ul className="actions__list">
                             <li className="actions__item">
                                 <a
+                                    id={`edit-link-${index}`}
```

Comment:
What's the reason for adding this ID?

Reply:
was experimenting will remove

Reply:
[fb7432e](https://github.com/fares-data-build-tool/create-data/pull/650/commits/fb7432ed8570a01866059022e52572386c96e304)